### PR TITLE
Move pangoText example from example script to Docs

### PR DESCRIPTION
**Pull request by kolibril13**
See title,
small change to make it more convenient to see this example

### CODE REVIEW

File: manim/mobject/svg/text_mobject.py [hunk 1]
```diff
@@ -598,7 +598,28 @@ class PangoText(SVGMobject):
...
+            def construct(self):
+                morning = PangoText("வணக்கம்", font="sans-serif")
+                chin = PangoText(
+                    "見 角 言 谷  辛 辰 辵 邑 酉 釆 里!", t2c={"見 角 言": BLUE}
+                )  # works same as ``Text``.
+                mess = PangoText("Multi-Language", style=BOLD)
+                russ = PangoText("Здравствуйте मस नम म ", font="sans-serif")
+                hin = PangoText("नमस्ते", font="sans-serif")
+                arb = PangoText(
+                    "صباح الخير \n تشرفت بمقابلتك", font="sans-serif"
```
Comment (kolibril13) [inline manim/mobject/svg/text_mobject.py:615]: This line \n  makes problems with building the docs.
Anyone an idea to solve this?
Maybe @behackl ?

Comment (behackl) [reply manim/mobject/svg/text_mobject.py:615]: Ahh, I see: the docstring should be marked as a raw string (add `r` before `"""` in Line585, https://github.com/ManimCommunity/manim/pull/533/files#diff-cf9247b89e1b4debba20823314a969f8R585). That should fix it. :-)

Comment (kolibril13) [reply manim/mobject/svg/text_mobject.py:615]: perfect!

**Review by leotrs — APPROVED** [review]

**Review by behackl — APPROVED** [review]
LGTM, thanks for moving this!

**Pull request merged**